_include/um/register/item_price_list.rst

```diff
@@ -0,0 +1,197 @@
+Medical Item Price Lists Administration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+  Pricelists of medical items are tools for specification which medical items and at which prices can be invoiced by contractual health facilities to the scheme administration. Administration of pricelists of medical items is restricted to users with the role of Scheme Administrator.
+
+Pre-conditions
+""""""""""""""
+
+  A price list of medical items may only be added, after an agreement with a health facility or a group of health facilities on specific prices. Editing of the price list may occur only after an approval of the management of the scheme administration. Deletion of a price list of medical Items normally will occur when a price list becomes obsolete.
```

Comment:
in first sentence 'medical items' is lowercase however in last sentence Items is uppercase